omnibus_overrides.rb

```diff
@@ -4,3 +4,6 @@
 train_stable = /^train \((.*)\)/.match(`gem list ^train$ --remote`)[1]
 override "train", version: "v#{train_stable}"
 override "ruby", version: "2.7.2"
+
+# Mac m1
+override "openssl", version: "1.1.1j" if mac_os_x? && arm?
```

Comment:
I feel like we would want to just update this for everything, no?

Reply:
InSpec has a FIPS build which requires 1.0.2. I'm not sure anyone is using the FIPS build, but it exists.

Reply:
We do not support FIPS on macOS platforms so this should not be an issue for us on macOS.

Reply:
I want to add some clarity for anyone passing by. There may be a little confusion here talking about Inspec FIPS builds versus Inspec macOS FIPS builds.

For all the products that have OpenSSL FIPS builds, we build those on RHEL + Windows only, and it's always been that way. There are customers/users using this functionality, we need to keep it, and it's why we've stayed with OpenSSL 1.0.2 so long.

Other platforms like macOS or Ubuntu do not have the necessary pieces in the Omnibus builds. If anyone is using FIPS on those other platforms they aren't doing it using our builds.

We've stayed on OpenSSL 1.0.2 on the non-FIPS platforms to maintain consistency, but we need a newer OpenSSL for macOS ARM. It makes sense if we're moving to 1.1.1 for macOS ARM to also switch to 1.1.1 for macOS x64 to maintain that platform consistency. I don't think we've justified yet moving all the other non-FIPS platforms to 1.1.1 yet, mostly because we don't have a need to yet.

Reply:
tldr; I support dropping the `&& arm?` part, but we need to keep the `if mac_os_x?` part.